test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -7,6 +7,7 @@ centos8 name=quay.io/ansible/centos8-test-container:1.21.0 python=3.6 seccomp=un
 fedora30 name=quay.io/ansible/fedora30-test-container:1.17.0 python=3.7
 fedora31 name=quay.io/ansible/fedora31-test-container:1.17.0 python=3.7
 fedora32 name=quay.io/ansible/fedora32-test-container:1.17.0 python=3.8
+fedora33 name=quay.io/ansible/fedora33-test-container:1.24.0 python=3.9
```

Comment:
Is it possible to also make use of the 3.10 alpha that's available in F33? There seems to be demand for that (https://github.com/ansible/ansible/pull/72852).

Reply:
Support for 3.10 is something that should be worked on separately from this PR.

Reply:
Totally, I just wanted to raise attention to that PR.